.travis/check-php-coding-style.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -o nounset
+set -o errexit
+
+if test "$TRAVIS_PULL_REQUEST:-" != 'false'; then
+    COMMIT_RANGE="$TRAVIS_COMMIT_RANGE"
+else
+    COMMIT_RANGE='HEAD~..HEAD'
+fi
+
+cd "$TRAVIS_BUILD_DIR"
+
+IFS='
```

Comment:
Does `IFS=$'\n'` also work? To make clear this is about a new line.

Reply:
Nope, `$'\n'` is a bash extension. This script (see the [shebang](https://github.com/concrete5/concrete5/blob/0ea880af2c4282bc812ff956fb18c02e1fcf5a8f/.travis/check-php-coding-style.sh#L1)) is meant to use whatever shell `/bin/sh` is.
This is the only POSIX-safe way I'm aware of (that is, it should work on every *nix shell).